<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memberViewReserves</class>
 <widget class="QMainWindow" name="memberViewReserves">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: #FFE5D0;
	font-family: Montserrat;
}

QLineEdit {
	border: 1px solid;
	border-radius: 15px;
	border-color: #e5e5e5;
	background-color: palette(base);
	padding: 0px 5px;
	font-size: 13px;
}

QPushButton {
	border-radius: 15px; 
	font-size: 13px;
	height: 31px; 
}</string>
   </property>
   <widget class="QLabel" name="pageTitle">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 24px;
	font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Reserved Books</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="loanedBooksIcon">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: rgb(231, 138, 107);
}</string>
       </property>
       <property name="text">
        <string>Catalogue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loanedBooksBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Loaned Books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reservedBooksIcon">
       <property name="text">
        <string>member</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reservedBooksBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Reserved Books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="editAccountIcon">
       <property name="text">
        <string>Reserved Books</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editAccountBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Account Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bcktoCatologueIcon">
       <property name="text">
        <string>back to catalogue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bcktoCatologueBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Back to Catologue</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="signoutIcon">
       <property name="text">
        <string>signOut</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signoutBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	border-radius:15px;
	font: 14px &quot;Montserrat&quot;;
}

QPushButton:hover {
	text-decoration: underline;
}</string>
       </property>
       <property name="text">
        <string>Sign Out</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pageDesc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Here are the books reserved under your account! These books will automatically be put in your account when they become available!</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/images/bblLogo.png</pixmap>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>1051</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
	border: none;
}
</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1051</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #FFE5D0;
	/*border: none;*/
}

QGridLayout {
	/*border: none;*/
}

QPushButton {
	margin-right: 10px; 
	height: 31px; 
	/*max-width: 100px; */
}
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_4"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>21</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line{
	color:rgb(221, 46, 68);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
